src/pages/iou/request/step/IOURequestStepDescription.js

```diff
@@ -60,6 +66,8 @@ function IOURequestStepDescription({
         }, []),
     );
 
+    const currentDescription = lodashGet(draftTransaction, 'comment.comment', undefined) || lodashGet(transaction, 'comment.comment', '');
```

Comment:
We should always use `transaction`. The only time we need to use `splitDraftTransaction` is if we are editing split request (due to its different logic flow)